reports/pvnet_comp.md

```diff
@@ -0,0 +1,269 @@
+
+### Mean Bias Error
+
+![Error Distribution of Model A](./imgs/pvnet_vs_pvlin_MBE_monthly.png)
+*Fig 7: Average monthly MBE for PVNet and PVLive Intraday*
+
+Looking at the Mean Bias Error across the months, PVLive Intraday has strong tendency to unpredicted, with this bias strongest in the summer months. PVNet also has a slight tendency to unpredict in these summer months as well before switching to having a slight tendency to overpredict in the winter. As the year progresses towards the winter months PVLive Intradays bias does start to decrease where it reaches around -60 in Novemeber compared to PVNet which sits around +50.
+
+Like before, the following heatmaps will break the MAE down into the average MAE per hour for months and weeks. With red showing an overprediction (the darker the red the greater the overprediction) and blue showing an underprediction (the darker the blue the greater the underprediction) for that period.
+
+<table>
+<tr>
+<td><img src="./imgs/pvlin_all_heatmap_MBE_Month.png"/></td>
+<td><img src="./imgs/pvnet_all_heatmap_MBE_Month.png"/></td>
+</tr>
+<tr>
+<td>Fig 8: PVLive Intraday MBE heatmap for Hour vs Month.</td>
+<td>Fig 9: PVNet MBE heatmap for Hour vs Month.</td>
+</tr>
+</table>
+
+<table>
+<tr>
+<td><img src="./imgs/pvlin_all_heatmap_MBE_week.png"/></td>
+<td><img src="./imgs/pvnet_all_heatmap_MBE_week_F.png"/></td>
+</tr>
+<tr>
+<td>Fig 10: PVLive Intraday MBE heatmap for Hour vs Week.</td>
+<td>Fig 11: PVNet MBE heatmap for Hour vs Week.</td>
+</tr>
+</table>
+
+The heatmaps shows that PVLive Intraday has a strong tendecy to underpredict with only 1 week having strong signs of an overprediction (week 35). PVNet illustrates and tendency to underpredict in the very early and later hours of the day (around sunrise and sunset) and slight tendency to overpredict in the middle of the day, but with some underpreictions occuring (primarily during week 30 to 34).
+
+### Error Distribution
+
+![Model Forecast vs. Actuals](./imgs/pvnet_pvlin_Error_nights_excluded_hist.png)
+*Fig 12: Histrogram of error distribution of PVNet and PVLive Inraday, with nights removed.*
+
+When viewing the error distrubtion, the effect of the mean bias in each of the forecasts becomes visible. With PVLive Intraday having a strong negative skew and PVNet having a symetrical distribution.
+
+---
+
+## PVNet Model Evaluation
+
+Next this report will look at the performance of PVNet compared to PVLive updated across 2022. PVNet was evaluated against PVLive updated, where data was present for each, over the period of:
+
+* 2022-01-01 to 2022-11-19
+
+The results shown in the table below are from taking that 0th hour forecast (now), which is the same that was used for the PVLive Intraday comparison above, and comparing the results to PVLive Updated. This section also contains graphs which illustrate the performance of PVNet across its different forecasting horizons (0-8hours).
```

Comment:
PVNet really predicts for the horizons (30mins- 8hours[480mins]). I think this slipped into the x-axis of some of your graphs as well

Reply:
Good spot, fixed